### ISSUE TITLE
local_user is nil in CircleCI

### ISSUE THREAD

**Issue by mochizukikotaro** [onthebeach/capistrano-slackify#34]
Hi!
I used capistrano-slackify in CircleCI 2.0, and encountered below error.

```
00:00 slack:notify_failed
      Notifying Slack of deploy failed
bundler: failed to load command: cap (/home/circleci/repo/vendor/bundle/ruby/2.4.0/bin/cap)
NoMethodError: undefined method `strip' for nil:NilClass
```

https://github.com/onthebeach/capistrano-slackify/blob/master/lib/capistrano/tasks/slackify.cap#L64

In CircleCI, `local_user` is `nil`. 
So, I added Environment Variable `USER` in CircleCI. But it was difficult to notice this cause. 

I am thinking of a pull request like below,

`capistrano-slackify/lib/capistrano/tasks/slackify.cap`

```diff
- set :slack_user, -> { local_user.strip }
+ set :slack_user, -> { local_user.to_s.strip }
```

what do you think about it? Please tell me any other good ideas. 
Thanks!

**Comment by seenmyfate** [comment]
Hi @mochizukikotaro, thanks for the report - that would be a good approach for me. Capistrano sets this value to [`ENV["USER"] || ENV["LOGNAME"] || ENV["USERNAME"]`](https://github.com/capistrano/capistrano/blob/aca431c91ea3baeaeb38665f365e73e9f73e1633/lib/capistrano/defaults.rb#L36) if it hasn't been explicitly set, so treating it always as string here seems like something that would inevitably trip us up at some point.

**Comment by mochizukikotaro** [comment]
Hi @seenmyfate, thanks!
I made pull request. https://github.com/onthebeach/capistrano-slackify/pulls

**Comment by seenmyfate** [comment]
@mochizukikotaro thanks for the quick turn around! This is now available on Rubygems as [v2.10.0](https://rubygems.org/gems/capistrano-slackify/versions/2.10.1)
